eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -213,6 +213,18 @@ jobs:
 
       - template: /eng/pipelines/templates/steps/show-failure-logs.yml
 
+#      - task: 1ES.PublishPipelineArtifact@1
```

Comment:
If there are plans to use in the future, include a comment about when it should be used. Looks to me like it's useful for getting build outputs for local inspection/debugging. 

Alternatively, it could be used as a `PreTestStep` in the event hubs' ci.yml to keep it out of a widely used template.

Reply:
That's exactly what this was used for - to export test binaries so they could be run locally.